.github/workflows/build-resources.yaml

```diff
@@ -0,0 +1,59 @@
+name: Build resources
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: "galasa version"
+                required: true
+            distBranch:
+                description: 'Destination branch'
+                required: true
+            ref:
+                description: 'Source branch'
+                required: false
+                default: 'main'
+
+env:
+    REGISTRY: ghcr.io
+    NAMESPACE: galasa-dev
+
+jobs:
+    branch-docker-build-resources:
+        runs-on: ubuntu-latest
+        steps: 
+            - name: Checkout resources in automation repository
+              uses: actions/checkout@v4
+              with:
+                ref: ${{inputs.ref}}
+                sparse-checkout: |
+                 dockerfiles/release/resources
+            
+            - name: Extract metadata (tags, labels) for Docker
+              id: meta
+              uses: docker/metadata-action@v5
+              with:
+                images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/galasa-resources:${{inputs.distBranch}}

```

Comment:
If you look at the Tekton resources-build, this is pushing the image to icr.io, not ghcr.io. However, I am fine for it to be moved to ghcr.io, but you'll also need to update the other parts of the release process that have the icr.io location (search the `/releasePipeline` directory in VSCode for the icr.io location)